docs/authentication/social-connections/linkedin-oidc.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: LinkedIn OpenID Connect
+description: Learn how to set up social connection with LinkedIn.
+---
+
+# LinkedIn
+
+How to set up social connection with LinkedIn OpenID Connect
+
+## Overview
+
+Adding social connection with LinkedIn to your app with Clerk is done in a few steps - you only need to set the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.
+
+To make the development flow as smooth as possible, Clerk uses preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed.
+
+For production instances, you will need to generate your own Client ID and Client secret using your LinkedIn account.
+
+> The purpose of this guide is to help you create a LinkedIn account and a LinkedIn OAuth app - if you're looking for step-by-step instructions using Clerk to add social connection (OAuth) to your application, follow the [Social connection (OAuth) guide](/docs/authentication/social-connections/oauth).
+
+## Before you start
+
+- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.dev/). For more information, check out our [Set up your application guide](/docs/quickstarts/setup-clerk).
+- You need to have a LinkedIn account. To create one, [click here](https://developer.linkedin.com/).
+
+## Configuring LinkedIn social connection
+
+First, you need to create a new OAuth LinkedIn app.
+
+![Creating an OAuth LinkedIn app](/docs/images/authentication-providers/linkedin/148540a6928a19c42d105d50c91bd4a61c1d3326-1096x978.png)
+
+You need to set a name, associate a LinkedIn page with it, and upload a logo for your new application. Go to the [Social Connections page](https://dashboard.clerk.dev/last-active?path=authentication/social) in your Clerk Dashboard and enable LinkedIn. In the modal that opened, ensure **Use custom credentials** is enabled and copy **Authorized redirect URI**. Paste the value into the **Redirect URL** of your LinkedIn app, as shown below.
+
+![Obtaining the Application ID and Client secret](/docs/images/authentication-providers/linkedin/linkedin1.png)
+
+The last thing you have to do is to enable the * Sign In with LinkedIn using OpenID Connect* product for your OAuth application.
```

Comment:
This should change to make it bold.

```suggestion
The last thing you have to do is to enable the **Sign In with LinkedIn using OpenID Connect** product for your OAuth application.
```

Reply:
🙃 Also, maybe change this to "step"?

```suggestion
The last step you have to do is to enable the * Sign In with LinkedIn using OpenID Connect* product for your OAuth application.
```

Reply:
Both of these are actually copy pasted from the deprecated page. I will let someone else say whether I should change these or not.